BookingsApi/BookingsApi/Controllers/HearingsController.cs

```diff
@@ -801,6 +799,36 @@ public async Task<IActionResult> GetUnallocatedHearings()
             return Ok(response);
         }
 
+        /// <summary>
+        /// Get hearing shell for a given hearing id
+        /// </summary>
+        /// <param name="hearingId">Id for a hearing</param>
+        /// <returns>Hearing shell</returns>
```

Comment:
I believe this might be a typo. Should it not be:

`<returns>Hearing status</returns>`